kong/plugins/azure-functions/handler.lua

```diff
@@ -61,17 +45,33 @@ function azure:access(config)
   local s = find(upstream_uri, "?", 1, true)
   upstream_uri = s and sub(upstream_uri, 1, s - 1) or upstream_uri
 
-  local path = conf.path
-  local end1 = path:sub(-1, -1)
-  local start2 = upstream_uri:sub(1, 1)
-  if end1 == "/" then
-    if start2 == "/" then
-      path = path .. upstream_uri:sub(2,-1)
+  local func = conf.functionname or ""
+  if func ~= "" then
```

Comment:
This and the `if` in line 54 below seem like uneccessary optimizations to me as well.